share/setup.nsi.in

```diff
@@ -80,8 +80,7 @@ Section -Main SEC0000
     File @abs_top_builddir@/release/@BITCOIN_CLI_NAME@@EXEEXT@
     File @abs_top_builddir@/release/@BITCOIN_TX_NAME@@EXEEXT@
     File @abs_top_builddir@/release/@BITCOIN_WALLET_TOOL_NAME@@EXEEXT@
-    SetOutPath $INSTDIR\doc
-    File /r /x Makefile* @abs_top_srcdir@/doc\*.*
```

Comment:
Do we not want to install *any* docs?

Reply:
what would be docs that would be critical to include with the installation executable?

Reply:
> Do we not want to install any docs?

Windows is the odd one out in this regard. I think the README and copyright is fine. I'd assume most users never even realised we were dumping a bumping of random/non-windows documentation onto their drives. If you think we should be installing some, then you should suggest a list.

Reply:
It's not a question of critical, just what's appropriate. I would think most of the docs would be. Exceptions are build docs, old release notes, and similar.

Reply:
>  I would think most of the docs would be. Exceptions are build docs, old release notes, and similar.

Again, feel free to propose a list / change. I think your assessment of "most" is incorrect. Of all the `.md` files in the 23.x `docs/` folder, less than half would be relevent.

In any case don't think this something worth spending time to try and optimise. If removing the installation of docs is too contreversial, happy to just drop that commit from this PR, and we can close the second half of #17171 as wont fix.